Clang error: Unknown type name 'nullptr_t'; did you mean 'std::nullptr_t'?

Today I've pulled a Selene and I've got this error:
```
In file included from /home/drmgc/sandbox/mmltest/src/game.cpp:7:
In file included from /home/drmgc/sandbox/mmltest/src/game.hpp:9:
In file included from /home/drmgc/sandbox/mmltest/3rd/selene/include/selene.h:7:
In file included from /home/drmgc/sandbox/mmltest/3rd/selene/include/selene/State.h:7:
In file included from /home/drmgc/sandbox/mmltest/3rd/selene/include/selene/Registry.h:3:
In file included from /home/drmgc/sandbox/mmltest/3rd/selene/include/selene/Class.h:3:
In file included from /home/drmgc/sandbox/mmltest/3rd/selene/include/selene/ClassFun.h:3:
In file included from /home/drmgc/sandbox/mmltest/3rd/selene/include/selene/BaseFun.h:3:
In file included from /home/drmgc/sandbox/mmltest/3rd/selene/include/selene/function.h:6:
/home/drmgc/sandbox/mmltest/3rd/selene/include/selene/references.h:64:28: error: C++ requires a type specifier for all declarations
    friend bool operator==(nullptr_t, Pointer<T> const & ptr) {
                           ^~~~~~~~~
/home/drmgc/sandbox/mmltest/3rd/selene/include/selene/references.h:68:52: error: unknown type name 'nullptr_t'; did you mean
      'std::nullptr_t'?
    friend bool operator==(Pointer<T> const & ptr, nullptr_t) {
                                                   ^~~~~~~~~
                                                   std::nullptr_t 
```

--- a/include/selene/references.h
+++ b/include/selene/references.h
@@ -61,19 +61,19 @@ public:
         return !_obj;
     }
 
-    friend bool operator==(nullptr_t, Pointer<T> const & ptr) {
+    friend bool operator==(std::nullptr_t, Pointer<T> const & ptr) {
         return nullptr == ptr._obj;
     }
 
-    friend bool operator==(Pointer<T> const & ptr, nullptr_t) {
+    friend bool operator==(Pointer<T> const & ptr, std::nullptr_t) {
         return nullptr == ptr._obj;
     }
 
-    friend bool operator!=(nullptr_t, Pointer<T> const & ptr) {
+    friend bool operator!=(std::nullptr_t, Pointer<T> const & ptr) {
         return !(nullptr == ptr);
     }
 
-    friend bool operator!=(Pointer<T> const & ptr, nullptr_t) {
+    friend bool operator!=(Pointer<T> const & ptr, std::nullptr_t) {
         return !(nullptr == ptr);
     }
 

